CONTRIBUTING.md

```diff
@@ -14,7 +14,63 @@ You can also reach out to Jeff on Twitter at [@tangjeff0](https://twitter.com/ta
 
 ## Development Environment
 
-See the setup in the README. (To be updated soon.)
+1. Download a `java` JDK. You can download the most [current version](https://www.oracle.com/java/technologies/javase-downloads.html) or access the [JDK archives](https://jdk.java.net/archive/).
+2. Download a package manager: [yarn](https://www.npmjs.com/package/yarn) or [npm](https://www.npmjs.com/get-npm). 
+3. Download and install the lein package manager (lein installs the correct version of Clojure for you). Detailed instructions for Linux, macOS, and Windows are below.
+
+### On Linux:
+   * Install curl command
+> ```sudo apt-get install -y curl```
+   * Download the lein script
+> ```curl https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein > lein```
+   * Move the lein script to the user programs directory
+> ```sudo mv lein /usr/local/bin/lein```
+   * Add execute permissions to the lein script
+> ```sudo chmod a+x /usr/local/bin/lein```
+   * Verify your installation
+> ```lein version```
+   
+   It should take a while to run, as it will download some resources it needs the first time. See the note at the end of this section if you are having issues.
+
+### On macOS:
+   * Download the lein script
+> ```curl https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein > lein```
+   * Move the lein script to the user programs directory
+> ```sudo mv lein /usr/local/bin/lein```
+   * Add execute permissions to the lein script
+> ```sudo chmod a+x /usr/local/bin/lein```
+   * Verify your installation
+> ```lein version```
+
+   It should take a while to run, as it will download some resources it needs the first time. See the note at the end of this section if you are having issues.
+
+### On Windows:
+   * Download the lein.bat script
+> ```curl -O https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein.bat```
+   * Create a bin directory for scripts
+> ```md bin```
+   * Move the lein.bat script to that directory
+> ```move lein.bat bin```
+   * Add bin to your path
+> ```setx path "%path%;%USERPROFILE%\bin"```
+   * Complete your installation
+   Close the command prompt and open a new one. Then run the following command to finish the installation.
+> ```lein self-install```
+   
+   It should take a while to run, as it will download some resources it needs the first time. See the note at the end of this section if you are having issues.
+
+4. Clone the repo found [here](https://github.com/athensresearch/athens). Change directory to the athens folder and run one of the following commands:
+> ```yarn install```
+
+> ```npm install```
```

Comment:
Let's stick to `yarn` for now. One option simplifies things. I've never had a problem going between the two in case we would ever need `npm`. Please also delete `package-lock.json`.

Reply:
@tangjeff0 Could you please bring `package-lock.json` back? It's possible to automate this using [synp](https://github.com/imsnif/synp). If there is a good reason to avoid NPM, I'd love to know why.